CONTRIBUTING.md

```diff
@@ -11,9 +11,10 @@ possible.
    and after the change. Examples of benchmarks can be found in `benchmarks/python/`.
 4. If you've changed APIs, update the documentation.
 5. Every PR should have passing tests and at least one review. 
-6. For code formatting install `pre-commit` using something like `pip install pre-commit` and run `pre-commit install`.
+6. For code formatting install `pre-commit`, `black` and `clang-format` using something like `pip install pre-commit black clang-format` and run `pre-commit install`.
```

Comment:
I don't think this is necessary. Could you revert it?

Reply:
Done